.github/workflows/prettier.yml

```diff
@@ -17,4 +17,6 @@ jobs:
           cache: "npm"
       - run: npm ci
       - name: Verify if all files are prettier-ed
-        run: npx prettier --check "**/*" --ignore-unknown
+        run: |
+          npx prettier --write ".husky/**/*"
```

Comment:
Sorry, I'm confused now. Why add this line? What's the purpose of writing in CI?

Reply:
To prettify the husky config/setup files after the npm install.

Reply:
OH I see what happened - sorry, I misinterpreted what you were saying!

> I guess we can prettify husky files in the CI 🤔 right ?

I thought you meant to confirm that it's ok for Prettier to _coincidentally_ write things. Like, if there's some other setup script already happening.

But, there's no need to _intentionally_ prettify husky files in CI. Users won't see it.

---

Basically, what I'm going for here is as little configurationc ode as possible. My proposal is:

* Keep this Prettier workflow to just running `npx prettier --check`
* Not add `.husky` to `.prettierignore`

Thoughts?

Reply:
that means I'll have to ignore it on the CI ? `npx prettier --check --ignore-path .husky/` ? Because the script would fail since we added prettier plugin for shell scripts.

Reply:
Ok, I think I see what you're saying. In that case you were write to `.prettierignore` the `.husky` directory... yes that makes sense!

Sorry, I was all over the place here 😄. But I think we're on the same page now! Let's `.prettierignore` `.husky` then.

Reply:
done 💪🏾 @JoshuaKGoldberg